### ISSUE TITLE
Java plugin not resolved as transitive dependency of Kotlin plugin

### ISSUE THREAD

**Issue by snrostov** [JetBrains/intellij-platform-gradle-plugin#655]
Adding `plugins.set(listOf("org.jetbrains.kotlin:211-1.4.21-release-IJ6693.43"))` causes
```
e: Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class org.jetbrains.kotlin.psi.KtFile, unresolved supertypes: com.intellij.psi.PsiClassOwner, com.intellij.psi.PsiModifiableCodeBlock
```

The example project in [attachment](https://github.com/JetBrains/gradle-intellij-plugin/files/6465108/untitled28.zip). `./gradlew build` for this project failed with the error above.

Adding `"java"` to `plugins` manually fixes the issue.

**Comment by YannCebron** [comment]
my 2ç: we shouldn't add transitive dependencies for _external_ plugins automatically

1) it will not work if the plugin is not bundled (need to specify version)
2) you might end up with unexpected huge dependency tree

we _could_ probably make an exception for bundled plugins, but these could change/become unavailable across different platforms/versions, so results are somewhat unpredictable again

**Comment by hsz** [comment]
This is now properly implemented in 2.0 — all transitive dependencies of the bundled plugins are included in the classpath. Note: Kotlin is a bundled plugin.
